e2e/commands/checkAccessibility.js

```diff
@@ -3,10 +3,15 @@ const { rootSelector } = require('../config')
 exports.command = function checkAccessibility() {
   this.initAccessibility().verify.accessibility(rootSelector, {
     verbose: false,
+    timeout: 10000,
```

Comment:
Marco: 'reduce it back to what it was'

Reply:
Nvm, this actually might be needed, misinterpreted what it did.